WCFDataService/Client/System/Data/Services/Client/Materialization/HttpWebResponseMessage.cs

```diff
@@ -98,6 +100,21 @@ internal HttpWebResponseMessage(HeaderCollection headers, int statusCode, Func<S
         }
 
         /// <summary>
+        /// Constructor.
+        /// </summary>
+        /// <param name="headers">The headers.</param>
+        /// <param name="statusCode">The status code.</param>
+        /// <param name="getResponseStream">A function returning the response stream.</param>
+        /// <param name="underlyingResponseMessage">The underlying response message that should be disposed together with this instance.</param>
+        internal HttpWebResponseMessage(HeaderCollection headers, int statusCode, Func<Stream> getResponseStream, IODataResponseMessage underlyingResponseMessage)
```

Comment:
getResponseStream and underlyingResponseMessage are dup. getResponseStream can be removed, and set this.getResponseStream to underlyingResponseMessage.getStream